migrations/versions/2018_11_30_fix_migrations_fix_migrations_by_bringing_in_.py

```diff
@@ -0,0 +1,72 @@
+"""Fix migrations by bringing in unaccepted changes
+
+Revision ID: 2018_11_30_fix_migrations
+Revises: 2018_11_28_drop_contact_details
+Create Date: 2018-12-02 13:27:43.384673
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "2018_11_30_fix_migrations"
+down_revision = "2018_11_28_drop_contact_details"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint("uq_classification_code", "classification", type_="unique")
+    op.create_foreign_key(
+        op.f("data_source_frequency_of_release_id_fkey"),
+        "data_source",
+        "frequency_of_release",
+        ["frequency_of_release_id"],
+        ["id"],
+    )
+    op.alter_column(
```

Comment:
@samuelhwilliams any idea why sqlalchemy wants to run this migration when there's already https://github.com/racedisparityaudit/ethnicity-facts-and-figures-publisher/pull/844/files#diff-2e189537477dbc38d5972e7ca4d0b3feR22 ?

Reply:
I think it's mostly about adding the `NOT NULL` constraint.

Reply:
This is outdated now - but it's because the later migration to add a `not null` constraint didn't also include the changes in Python code to tell SQLAlchemy to expect it to have that constraint.

Reply:
@samuelhwilliams ah ok, that's my fault. I didn't twig that SQLAlchemy needed to be told about that (as Rails works slightly differently).